doc/10_GraphQL/04_Query/11_Query_Samples/11_Sample_GetAsset.md

```diff
@@ -54,15 +55,15 @@ Note that for the fullpath and the base64 encoded data you can specify a thumbna
             "getAsset": {
               "id": "4",
               "fullpath": "/Car%20Images/jaguar/auto-automobile-automotive-192499.jpg",
-              "assetThumb": "/Car%20Images/jaguar/image-thumb__4__exampleCover/auto-automobile-automotive-192499.webp",
-              "assetThumb2": "/Car%20Images/jaguar/image-thumb__4__content/auto-automobile-automotive-192499.webp",
+              "assetThumb": "/Car%20Images/jaguar/4/image-thumb__4__exampleCover/auto-automobile-automotive-192499.jpg",
```

Comment:
Is this a BC break as now it returns a different format when no format is provided?

Reply:
I think it is fine, because it changed in pimcore itself (in 2021 already), like Bernie mention here (https://github.com/pimcore/pimcore/discussions/10430#discussioncomment-1396465) . And the data-hub´s behaviour didn´t change with the pr, I just adapted the documentation.